package.json

```
@@ -3,6 +3,7 @@
     "private": true,
     "scripts": {
         "create-site-configs-env": "npx @comet/cli inject-site-configs -f site-configs/site-configs.ts -i .env.site-configs.tpl -o .env.site-configs",
+        "build-and-run-site": "./build-and-run-site.sh; npx dev-pm start site",
```

```suggestion
        "build-and-run-site": "./build-and-run-site.sh; npx dev-pm restart site",
```

Should this be restart? What if I run this script, but the site wasn't running before?